docs/build-tools/gradle.md

```diff
@@ -93,5 +93,5 @@ plugin in Bloop config.
 
 This is much more complex than in case of the automatic installation, so it is
 recommended to only do manual installation when having problems with the
-automatic one. You can also always try to reach us on the Metals gitter channel
+automatic one. You can also always try to reach us on the [Metals gitter channel](url)
```

Comment:
typo? This should be `https://gitter.im/scalameta/metals`

Reply:
Right, I just commited Marek's changes without looking exactly.